ApplensBackend/Services/DiagnosticRoleClientService/DiagnosticRoleClient.cs

```diff
@@ -66,30 +66,28 @@ private HttpClient InitializeClient()
                 MaxResponseContentBufferSize = Int32.MaxValue
             };
 
-            client.DefaultRequestHeaders.Add("internal-applens", "true");
+            client.DefaultRequestHeaders.Add("x-ms-internal-client", "true");
```

Comment:
>x-ms-internal-client [](start = 46, length = 20)

Any specific reason we changed this from internal-applens? Is this some kind of convention that we are trying to follow ?